frontend/src/components/print/print-react/documents/campPrint/renderPdf.worker.js

```diff
@@ -18,6 +19,11 @@ self.start = (...args) => {
 }
 
 export const renderPdfInWorker = async (data) => {
+  if (data?.config?.language) {
```

Comment:
Shouldn't the language property be always set?
Then we could throw an error if not.

Else we have the same behavior as before. Maybe it resolves itself after some time... but maybe an error in sentry is also helpful

Reply:
Removing the if will not throw an error if config is defined, but language is not. (and you set the language again to undefined, which does not help if dayjs does not have a guard.)
No strong oppinion, but i would throw an Error (IllegalArgumentExcpetion for java speaking people :wink: )